media/plg_system_tour/js/guide.js

```diff
@@ -69,7 +69,7 @@ Joomla = window.Joomla || {};
                     },
                     onError : function() {
                         Joomla.request({
-                            url: 'https://yveshoppe.de/jdocsapi/' + urlParameters.langtag.split('-')[0]  + '/' + 'jsonNotFound.json' ,
+                            url: 'https://yveshoppe.de/jdocsapi/' + 'en-US'  + '/' + 'jsonNotFound.json' ,
```

Comment:
Not en-GB?

Reply:
@yvesh en-GB directory is not available over the Crowdin. As it is default in Joomla. So I have to direct to en-US.

Reply:
but it later will be, when we do a mapping. @Sandra97 do we separate between en-US / en-GB in crowdin?

Reply:
`en-GB` is always the source language so there's no 'folder' for it, and no reason to have it over Crowdin.
`en-US` is just a language like any other ones, fr-FR or whatever. 


.

Reply:
@Sandra97 yeah, but we won't deliver any language with joomla itself. So we need en-GB in the CDN too?! And this should be the default?

Reply:
Even if you don't deliver any language, I presume you'll have some text content in the tour? So yes, en-GB. en-GB is always the default langauge.